README.md

```diff
@@ -89,6 +90,56 @@ Example `.eslintrc.json` for a **TypeScript React** project:
 }
 ```
 
+Example `.eslintrc.json` for a **Vue 3** project:
+
+```json
+{
+  "root": true, 
+  "extends": [
+    "hardcore",
+    "hardcore/ts-for-js",
+    "hardcore/vue"
+  ], 
+  "parserOptions": {
+    "parser": "@typescript-eslint/parser", 
+    "project": "tsconfig.json"
+  },
+  "settings": {
+    "import/resolver": {
+      "alias": {
+        "map": [["@", "./src/"]], 
+        "extensions": [".js", ".vue"]
+      }
+    }
+  }
+}
+```
+<sub>Make sure to also add **eslint-import-resolver-alias** to your project.<br>If you would like to use **ts-for-js**, do not forget to create a **tsconfig.json** file containing ```{}```. If you would not like to use it, you can omit the **parserOptions** section. </sub>
```

Comment:
Is there a way to include eslint-import-resolver-alias in eslint-config-hardcore in a generic way, so it would work for most Vue 3 projects? The goal is to make eslint-config-hardcore work with as little effort as possible.

Reply:
Yes, that should work, just like eslint-import-resolver-typescript.